examples/hypervisor_cluster_profile.tf

```diff
@@ -0,0 +1,96 @@
+provider "oneview" {
+	ov_username = "Administrator"
+	ov_password = "admin123"
+	ov_endpoint = "https://10.50.9.41"
+	ov_sslverify = false
+	ov_apiversion = 1600
+	ov_ifmatch = "*"
+}
+
+data "oneview_scope" "scope_obj" {
+        name = "test_scope"
+}
+
+resource "oneview_hypervisor_cluster_profile" "HypervisorClusterProfile"{  
+    "type"="HypervisorClusterProfileV3",
+    "name"="Cluster",
+    "description"="asa",
+    "hypervisor_type"="Vmware",
+    "hypervisor_manager_uri"="/rest/hypervisor-managers/4d47b4ca-1c40-47ad-b170-099b04e3691f",
+    "hypervisor_cluster_settings"={  
+                                  "type"="Vmware",
+                                  "drs_enabled"=true,
+                                  "ha_enabled"=false,
+                                  "multi_nic_v_motion"=false,
+                                  "virtual_switch_type"="Standard"
+                               },
+    "hypervisor_host_profile_template"={  
+                                  "server_profile_template_uri"="/rest/server-profile-templates/87afd6f4-8b24-4182-8f06-8b5cc5b40786"
+                                  "host_prefix"="Cluster",
+
+     }
+}
+/*
+
```

Comment:
Comment saying that this resource body is for update and needs to be uncommented when used.

Reply:
done